src/sentry/search/events/builder/base.py

```diff
@@ -72,18 +72,20 @@
     raw_snql_query,
     resolve_column,
 )
+from sentry.utils.validators import INVALID_ID_DETAILS, INVALID_SPAN_ID, WILDCARD_NOT_ALLOWED
 
 
 class BaseQueryBuilder:
     requires_organization_condition: bool = False
     organization_column: str = "organization.id"
-    uuid_fields = {"id", "trace", "profile.id", "replay.id"}
     function_alias_prefix: str | None = None
     spans_metrics_builder = False
     profile_functions_metrics_builder = False
     entity: Entity | None = None
     config_class: type[DatasetConfig] | None = None
     duration_fields: set[str] = set()
+    uuid_fields: Mapping[str, str] = {}
```

Comment:
Nice, i like that you made the value a label for more human readable messages, do you mind commenting that here on the base class too?

Reply:
On second thought, the final error message does read as smoothly like this.

```python
"Wildcard conditions are not permitted on `trace.span` field"
```

vs

```python
"Wildcard conditions are not permitted on Filter Trace ID field"
```

Any thoughts on switching it just showing the column name surrounded in backticks?

Reply:
Ah, that's a very good point. I think customers will still be able to figure it out either way, but its probably easier for both sides if we stick with the column name